dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -42,25 +42,25 @@ public static void prepare() throws Exception {
     public void Test_Store_Json_Then_Recover_Decrypted_Then_Delete_Then_Verify_Its_Gone()
             throws DotDataException, DotSecurityException {
         final User admin = TestUserUtils.getAdminUser();
-        ServiceSecrets.Builder builder = new ServiceSecrets.Builder();
-        final String serviceKey = "anyServiceKey";
-        final ServiceSecrets bean = builder.withServiceKey(serviceKey)
+        AppSecrets.Builder builder = new AppSecrets.Builder();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'builder' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)